src/rules/function-space-after/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  ruleMessages,
+  isWhitespace
+} from "../../utils"
+
+export const ruleName = "function-space-after"
+
+export const messages = ruleMessages(ruleName, {
+  expected: "Expected single space after function's \")\"",
+  rejected: "Unexpected space after function's \")\"",
+})
+
+const acceptableNextChars = [ " ", ")", undefined ]
+
+/**
+ * @param {"always"|"never"} expectation
+ */
+export default function (expectation) {
+  return function (css, result) {
+    css.eachDecl(function (decl) {
+      if (decl.prop === "content") { return }
+
+      const value = decl.value
+
+      for (let i = 0, l = value.length; i < l; i++) {
+        if (value[i] === ")") {
```

Comment:
Hadn't thought about that --- yeah, maybe balanced-match would be important, to make sure we get stuff like that and also make sure that we ignore things within strings (e.g. `content: "look at this attr(data-whatever)"`). Could you make an issue for that?